recipes-connectivity/libconnman-qt/libconnman-qt5/0001-Add-missing-declarations-for-operator-overloads.patch

```diff
@@ -17,14 +17,12 @@ Signed-off-by: Khem Raj <raj.khem@gmail.com>
 
 --- a/libconnman-qt/marshalutils.cpp
 +++ b/libconnman-qt/marshalutils.cpp
-@@ -36,6 +36,11 @@
+@@ -36,6 +36,9 @@
  
  #include "marshalutils.h"
  
-+#if defined(__clang__) &&  __clang_major__ >= 11
 +Q_DBUS_EXPORT const QDBusArgument &operator>>(const QDBusArgument &a, RouteStructure &v);
 +Q_DBUS_EXPORT QDBusArgument &operator<<(QDBusArgument &a, const RouteStructure &v);
-+#endif
```

Comment:
This one doesn't seem to be compatible with gcc-11 currently used in oe-core/master

http://errors.yoctoproject.org/Errors/Details/655056/

Reply:
now gcc 12 is default in core/master